src/actions/ScanActions.tsx

```diff
@@ -175,7 +175,7 @@ export function handleWalletUris(
   fioAddress?: string
 ): ThunkAction<Promise<void>> {
   return async (dispatch, getState) => {
-    const { legacyAddress, metadata, minNativeAmount, nativeAmount, publicAddress, uniqueIdentifier, tokenId } = parsedUri
+    const { legacyAddress, metadata, minNativeAmount, nativeAmount, publicAddress, uniqueIdentifier, tokenId = null } = parsedUri
```

Comment:
A note about this one: I made EdgeParsedUri.tokenId optional, since it doesn't apply in all scenarios. For instance, something like a bitId URI would not include a currency code / tokenId at all.

More importantly, we would need to add upgrade logic into the core, in case an older currency plugin fails to pass tokenId, we can still derive it from the currency code. We don't want illegal values slipping past the type system, since that's a recipe for crashes.

I did not want to enter this minefield, so I'm leaving the status quo for now.

The correct fix is to make EdgeParsedUri a union type, reflecting the different types of things we might scan (payments / private keys / wallet connect / bitId / request for address / etc.), so we can make tokenId mandatory in the places where it matters.